Source/WebKit/UIProcess/ios/fullscreen/WKFullScreenWindowControllerIOS.mm

```diff
@@ -1562,6 +1586,15 @@ - (void)_configureSpatialFullScreenTransition
     });
 }
 
+- (void)_setOrnamentsVisibility:(BOOL)visible
+{
+    WKFullScreenParentWindowState *originalState = _parentWindowState.get();
```

Comment:
This variable is unnecessary, you can just use square bracket syntax with `originalState`.